### PR TITLE
Val2023 04 03 synf usa

### PR DESCRIPTION
**Pull request by robertpietzcker**
## Purpose of this PR

1. update synfuel costs to make them more in line with other conversion tech costs in REMIND. This also prevents the gas->H2->MeOH bubble that in fact should be gasftrec in scenarios with low carbon prices.  
2. upper bound to USA 2025 PV deployment

## Type of change

- [x] Bug fix 
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)

## Further information (optional):

* Test runs are here: /p/projects/remind/users/robertp/save_for_pullRequests/2023-04-03_Validation_WithSynfUSA
(runs with "_USAsynf" contain the changes for this PR)
* Comparison of results (what changes by this PR?): 
synfuel use from the gas2h2 -> MeOH bubble in MEA and other regions in NPI and NDC disappeared - see https://github.com/remindmodel/development_issues/issues/194

### CODE REVIEW

File: core/datainput.gms [hunk 1]
```diff
@@ -462,6 +462,11 @@ $offdelim
 $Onlisting
 $ENDIF.WindOff
 
+*** test-add USA cap. Forecast by Wood Mackenzie Solar Market Insight Report 2022 ~ 265 GW DC in 2025 in bullish scenario
+!! Should be less in GW_AC, but REMIND corrects for lower model CF than real world by upscaling capacity - should be roughly ok.
+pm_histCap("2025","USA","spv") = 0.265;
```
Comment (LaviniaBaumstark) [inline core/datainput.gms:467]: this part was only for test, right? should be deleted also as it includes a specific region

**Review by LaviniaBaumstark — CHANGES_REQUESTED** [review]

File: core/datainput.gms (hunk 1 above)
Comment (robertpietzcker) [reply core/datainput.gms:467]: yes, totally - this was only there to be able to test it before the new input data is available. 
will delete

**Review by LaviniaBaumstark — APPROVED** [review]

**Comment by nicobauer** [comment]
Robert, I do not understand how the MEA GTL-problem is solved by limiting US SPV capacity.

**Comment by robertpietzcker** [comment]
> Robert, I do not understand how the MEA GTL-problem is solved by limiting US SPV capacity.

sorry - these are 2 different commits merged in one go. 

The main change is the synfuel one (described under "1." in the PR) - so for you, @nicobauer , only the changes to generisdata_tech.prn are relevant. :-)

the other one is a tiny change limiting unrealistic PV deployment in the US in the next 2 years that Christoph asked me to do.

**Review by PhilippVerpoort — APPROVED** [review]

**Pull request merged**